Tasks/Gradle/CodeAnalysis/Common/BuildOutput.ts

```diff
@@ -59,14 +54,12 @@ export class BuildOutput {
     private getBuildDirectoryName(): string {
         switch (this.buildEngine) {
             case BuildEngine.Gradle:
-                {
-                    return 'build';
-                }
+                return 'build';
```

Comment:
Not a problem with this refactoring exercise, but it seems wrong to have this code here.  

This file is under 'Gradle' folder, if it's truly common code, we should move it to 'Common'.  As it stands today, it appears we have duplicated copy of the same code?